lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -4,28 +4,40 @@ LwM2M carrier
 #############
 
 You can use the **LwM2M carrier** library in your nRF91 application in order to connect to the operator LwM2M network.
-Integrating this library into an application will cause the device to connect to the device management platform autonomously.
+Integrating this library into an application causes the device to connect to the device management platform autonomously.
+
 Every released version is considered for certification with carriers.
-Refer to the release notes for each version of the library to check the certification status of a particular release.
-Your final device must go through certification by Verizon Wireless.
+Refer to the :ref:`liblwm2m_carrier_changelog` to check the certification status of a particular release.
+Your final device must go through certification by the carrier.
+
+The :ref:`lwm2m_carrier` sample demonstrates how to run this library in an application.
 
 Prerequisites
 *************
-The LwM2M carrier library has dependencies on Zephyr and the nRF Connect SDK.
-Check the release notes for the library to see which version of |NCS| it is built for.
-To integrate the library in an application, the application must be based on a supported nRF Connect SDK version.
+The LwM2M carrier library has dependencies on Zephyr and the |NCS|.
+See the :ref:`liblwm2m_carrier_changelog` of the library to check which version of |NCS| it is built for.
```

Comment:
Does the changelog has this information? I only see the modem firmware version, not the NCS version.

Reply:
Added in the changelog